### PR TITLE
Add d3-cloud w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10805

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/jasondavies/d3-cloud
 * Official website (optional, not the repository):  https://www.jasondavies.com/wordcloud/
 * License and its reference: [BSD-3-Clause ](https://github.com/jasondavies/d3-cloud/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 122
   - Count of stars: 1,999
   - Count of forks: 725

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/d3-cloud/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "version": "1.2.4",
+  "description": "Generate word clouds in JavaScript.",
+  "keywords": [
+    "word",
+    "cloud",
+    "tag",
+    "visualization",
+    "canvas"
+  ],
+  "homepage": "https://www.jasondavies.com/wordcloud/",
```
Comment (pvnr0082t) [inline ajax/libs/d3-cloud/package.json]: I think it's just a demo page. We should remove it.

Comment (extend1994) [inline ajax/libs/d3-cloud/package.json]: @pvnr0082t I reserved it because it's in the original `package.json`. Should I still remove it?

Comment (pvnr0082t) [inline ajax/libs/d3-cloud/package.json]: Okay, let's keep it.

**Comment by pvnr0082t** [comment]
@extend1994 Please rebase onto the latest master branch. Thanks.

**Comment by extend1994** [comment]
@pvnr0082t I rebased the branch, please help me review it, thanks!

File: ajax/libs/d3-cloud/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/d3-cloud/package.json]: If it's really unlike a homepage, we can still remove it.

**Pull request merged**